server/rpc/interceptors/context.go

```diff
@@ -34,15 +39,24 @@ import (
 	"github.com/yorkie-team/yorkie/server/rpc/metadata"
 )
 
+const projectInfoCacheSize = 1
```

Comment:
It looks like the size of the cache is 1, so the cache only holds one project in a time. I wonder if it is intended.

Reply:
Yes it's intended, I understood that a each project can have one ProjectInfo, isn't it?

Reply:
No it's not intended. I had it set to 256, but I think I accidentally changed it. 

and here's something I'd like to discuss. how much CacheSize and TTL are appropriate?

I was going to set the CacheSize to 256 and the TTL to 1 hour, but what do you think?

Reply:
I also think 256 and 1 hour are suitable for the current situation. This is because yorkie.dev doesn't have many projects yet and project updates don't affect requests from clients.